resources/install/debian/copyright

```diff
@@ -1,126 +1,13 @@
-Format: http://dep.debian.net/deps/dep5
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 Upstream-Name: Jitsi Videobridge
 Upstream-Contact: Emil Ivov <emcho@jitsi.org>
 Source: https://github.com/jitsi/jitsi-videobridge
 
 Files: *
-Copyright: 2015 Atlassian Pty Ltd
+Copyright: 2015-2018 Atlassian Pty Ltd
```

Comment:
How about 2015-present ? We’ve been using that format in Java and ObjC files on mobile, FWIW.